wordle-web/components/game-board.vue

```diff
@@ -1,13 +1,28 @@
 <template>
-  <v-card width="450">
+  <v-card class="ma-5 py-1" max-width="500">
```

Comment:
There is supposed to be a gradient background here for the game board. try using `style="gradient code here"`